contest/contest_info.py

```diff
@@ -97,10 +99,13 @@ def get_scoreboard(contest):
 
     return scoreboard
 
-def get_clarifications(contest,user):
-    reply_all = Clarification.objects.filter(contest = contest, reply_all = True)
-    user_ask = Clarification.objects.filter(contest = contest, asker = user)
-    return reply_all | user_ask
+def get_clarifications(user,contest):
```

Comment:
Sorry, I forgot that.
Then, you should check a user is authenticated before getting user_ask list.